### PR TITLE
Redesigned /partners index

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- Updated /partners to match the design
- Updated top navigation to use the new logo, search and a dropdown for /partners pages

## QA

- View the site in your web browser at: https://canonical-com-521.demos.haus/partners
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches the [design](https://app.zeplin.io/project/624ed20e0af92866116ea26d/screen/625418358b950467d5a7f9e2)

## Issue / Card

Fixes [#5076](https://github.com/canonical-web-and-design/web-squad/issues/5076)

### CODE REVIEW

**Comment by spencerbygraves** [comment]
Thanks @sowasred2012 a couple of things...

- For the text in the partners tag (need to think of a name for this thing) I'd used the table heading size which is a bit bigger, and can we make the line 32px high please
- Also, I think that strip would be better with `is-shallow` please

Thanks

**Comment by spencerbygraves** [comment]
@sowasred2012 here are the icons and Suru backgrounds to try. Thanks.

**Icons**

- [Public Cloud](https://assets.ubuntu.com/v1/37b10bed-canonical-partners-public-cloud.svg)
- [IHV/OEM](https://assets.ubuntu.com/v1/42490eb8-canonical-partners-ihv-oem.svg)
- [Desktop](https://assets.ubuntu.com/v1/958544ed-canonical-partners-desktop.svg)
- [Channel reseller](https://assets.ubuntu.com/v1/1692cb51-canonical-partners-channel.svg)
- [IoT](https://assets.ubuntu.com/v1/3757b08f-canonical-partners-IoT.svg)
- [GSI](https://assets.ubuntu.com/v1/bce54960-canonical-partners-GSI.svg)

**Suru**

- [Light grey](https://assets.ubuntu.com/v1/1712bdc6-suru-2.0-light-1.jpg)
- [Dark grey](https://assets.ubuntu.com/v1/70b49764-suru-2.0-dark-1.jpg)

File: templates/partial/_footer.html [hunk 1]
```diff
@@ -1,38 +1,50 @@
-<footer class="p-footer is-dark">
-    <div class="row">
-      <hr />
+<footer class="p-strip--suru-dark-grey">
```
Comment (minkyngkm) [inline templates/partial/_footer.html:1]: The bullets look a bit weird when the screen gets smaller.
image

**Review by minkyngkm — COMMENTED** [review]
Looking great @sowasred2012! I just made some comments for minor things!

File: templates/partial/_footer.html (hunk 1 above)
Comment (sowasred2012) [reply templates/partial/_footer.html:1]: I just checked, that's a bug in Vanilla - I'll file an issue!

Comment (sowasred2012) [reply templates/partial/_footer.html:1]: Issue filed: https://github.com/canonical-web-and-design/vanilla-framework/issues/4429

File: templates/partial/_footer.html [hunk 2]
```diff
@@ -1,38 +1,50 @@
...
-        <p>
-          Ubuntu and Canonical are registered trademarks.<br />
-          All other trademarks are the property of their respective owners.
-        </p>
-      </div>
-    </div>
-  </footer>
+  </div>
+
+  <div class="u-fixed-width">
```
[the remark this answers is not in the document]
Comment (sowasred2012) [reply templates/partial/_footer.html:22]: Nice, will do

File: templates/partners/index.html [hunk 3]
```diff
@@ -1,76 +1,120 @@
...
+        alt="",
+        width="48",
+        height="48",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+      <h2 class="p-heading--3"><a href="/partners/public-cloud">Public Cloud&nbsp;&rsaquo;</a></h2>
+        
+      <p>Ubuntu on clouds is optimised, tested and certified by Canonical for performance and security</p>
```
[the remark this answers is not in the document]
Comment (sowasred2012) [reply templates/partners/index.html:38]: Yeah, you're right, it does look better :+1:

**Review by minkyngkm — APPROVED** [review]
LGMT! Thanks Scott!👍

**Pull request merged**